python/RDS_ENHANCED_MONITORING_ENABLED/RDS_ENHANCED_MONITORING_ENABLED_test.py

```diff
@@ -0,0 +1,195 @@
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import MagicMock
+import json
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::RDS::DBInstance'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+
+class Boto3Mock():
+    @staticmethod
+    def client(client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        if client_name == 'sts':
+            return STS_CLIENT_MOCK
+        raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('RDS_ENHANCED_MONITORING_ENABLED')
+
+class ComplianceTest(unittest.TestCase):
+    #scenario1
+    rule_invalid_parameter = '{"monitoringInterval":"12"}'
+    #scenario2,scenario5
+    rule_valid_parameter = '{"monitoringInterval":"5"}'
+    #scenario3
+    rule_parameter_mismatch = '{"monitoringInterval":"10"}'
+
+    valid_em_interval_configured = {
+        "monitoringInterval": "5"
+    }
+
+    invalid_em_interval_zero = {
+        "monitoringInterval": "0"
+    }
+
+    def test_scenario_1_invalid_parameter_value(self):
+        invoking_event = generate_invoking_event(self.valid_em_interval_configured)
+        response = RULE.lambda_handler(
+            build_lambda_configurationchange_event(invoking_event, rule_parameters=self.rule_invalid_parameter), {})
+        assert_customer_error_response(self, response, 'Invalid value for the parameter "monitoringInterval", Expected a valid integer from the list [1, 5, 10, 15, 30, 60].')
+
+    def test_scenario_2_interval_zero(self):
+        invoking_event = generate_invoking_event(self.invalid_em_interval_zero)
+        response = RULE.lambda_handler(
+            build_lambda_configurationchange_event(invoking_event, rule_parameters=self.rule_valid_parameter), {})
+        assert_successful_evaluation(self, response, [build_expected_response('NON_COMPLIANT', '123456789012', annotation="Enhanced Monitoring interval for this Amazon RDS instance is not configured.")])
```

Comment:
resource Id '123456789012' appears like an account id, we do use pseudo values but something like test-instance seems more relative.